packages/web3-eth2-core/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "web3-eth2-core",
+    "version": "1.3.0",
+    "description": "Web3 ETH2 core tools for sub-packages. This is an internal package.",
+    "repository": "https://github.com/ethereum/web3.js/tree/1.x/packages/web3-eth2-core",
+    "license": "LGPL-3.0",
```

Comment:
I think this has to change to MIT, should match repo licensing

Reply:
Copied from other packages again, I think all the packages have `LGPL-3.0` as their license, should we create an issue to update this across the entire project?

Reply:
nvm we good